.travis.yml

```diff
@@ -126,20 +126,3 @@ jobs:
     - script: unbuffer node build-system/pr-check/performance-tests.js # See #28148
     - script: unbuffer node build-system/pr-check/remote-tests.js # See #28343
   fast_finish: true
-before_cache:
-  # do not store cache for pr builds or experiment stage builds
-  - if [[ $TRAVIS_EVENT_TYPE == pull_request ]] || [[ $TRAVIS_BUILD_STAGE_NAME == experiment ]]; then exit $TRAVIS_TEST_RESULT ; fi
-cache:
```

Comment:
With this, we can also delete all the `CACHE_NAME` environment settings across this file. (They were used to separate caches from different jobs.)

Reply:
Done